### PR TITLE
Add Xin Huang profile

### LINKED ISSUE #2553: [Community] Member Profile: Xin Huang
**Current Behavior**

@gyohuangxin has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @gyohuangxin as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @gyohuangxin
- Twitter: @gyohuang
- LinkedIn: https://www.linkedin.com/in/Xin-Huang
- Link to profile picture: 

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by maneeshsagar**
Signed-off-by: Maneesh Sagar <maneeshsagar97@gmail.com>

**Description**
Added the details Xin Huang.
This PR fixes #2553 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/members/xin-huang/index.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+position: Contributor
+image_path: ./xin-huang.jpeg
+github: gyohuangxin
+twitter: gyohuang
+linkedin: Xin-Huang
+location: Shanghai, China
+bio: I am a cloud software developer from Intel. I focus on the acceleration on open source softwares for service mesh, such as Istio, Envoy. And I am also a contributor of Service Mesh Performance and the meshery-perf component. I love the open culture of Layer5, where you can find everything about open source and service mesh, and you will love it too.
+status: Active
+published: true
+---
```
Comment (adithyaakrishna) [inline src/collections/members/xin-huang/index.mdx:12]: @gyohuangxin Could you please confirm these regarding your profile?

File: src/collections/members/xin-huang/index.mdx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+name:       Xin Huang
```
Comment (verma-kunal) [inline src/collections/members/xin-huang/index.mdx:2]:
```suggestion
name: Xin Huang
```

Comment (verma-kunal) [reply src/collections/members/xin-huang/index.mdx]: unnecessary space here can be removed

**Comment by verma-kunal** [comment]
Hey @maneeshsagar , you would have to sign the new commit that you made!
Refer this to know more how to do this: https://github.com/layer5io/layer5/pull/2555/checks?check_run_id=4989933189

**Review by gyohuangxin — APPROVED** [review]
LGTM, thanks!

**Review by verma-kunal — APPROVED** [review]
Great work! 👍

**Comment by iambami** [comment]
Thank you @maneeshsagar 🤗

Good way to make your first contribution.😁

**Comment by l5io** [comment]
🚀 Preview for commit 754546fae8bc5bae2031d9eb3d33d91e3fe352f5 at: https://61f526a5e3210909b41bbe62--layer5ng.netlify.app

**Review by debo19 — APPROVED** [review]

**Pull request merged**